### ISSUE TITLE
WINE setups with splitted installers fail

### ISSUE THREAD

**Issue by sixsixfive** [lutris/lutris#328]
[Painkiller](https://lutris.net/games/install/2512/view) from GOG.com comes in multiple files( setup_painkiller_black_2.1.0.9.exe, setup_painkiller_black_2.1.0.9-1.bin)

Lutris cant find the second file

sure i could name the second file: setup_painkiller_black_2.1.0.9-1.bin but this will break with every version update...

**Comment by sixsixfive** [comment]
sorry false alarm, the installer works, however lutris fails/sleeps if there is no free disk space available(there should be a warning if the gamefolder runs out of disk space)

**Comment by strycore** [comment]
I started investigating on this and I found some issues, at least on the git master branch. The installer would not make any progress with your version so I forked to test more things. I removed the silent flags on the GOG installer and the game installed but the wine process would  not quit after it was done, requiring it to be manually killed.

I have to test this on Lutris 0.3.8 to see if it's an actual regression or if something else is going wrong.
